build.gradle

```diff
@@ -133,6 +133,10 @@ subprojects { subproject ->
 		targetCompatibility = 1.8
 	}
 
+	compileTestJava {
+		sourceCompatibility = 11
```

Comment:
What made you to move to Java 11?
Thanks

Reply:
The main driver was `Map.of(new TopicPartition("foo", 0), 0L, new TopicPartition("foo", 1), 0L)` in tests. I hate building maps in tests with `put()`; that's why I added `MapBuilder` in SI and SA (but we don't have one in SK).

But we also get new language features (`var`) - like we did with SI 4.3.x and using Java 8 in tests.

Reply:
Sure! Just curious what could be so useful in other projects.

We might need re-test the whole project to be sure that JavaDocs, for example, are built OK, too.

Reply:
I actually went to 14 initially, but travis only goes up to 11 😦

Reply:
Good to know. Merging...

Reply:
Ugh... https://stackoverflow.com/questions/51992347/missing-iframe-view-for-javadocs-jdk-11

Reply:
I guess we'll get used to it.